radix-transaction-scenarios/src/runners/dumper.rs

```diff
@@ -78,11 +78,11 @@ pub fn run_all_in_memory_and_dump_examples(
 
     assert_eq!(
         substate_db.get_current_root_hash().to_string(),
-        "43be4cce2d4f2ed2eb519d77dfa770697244e843b2a0f7fd86bdf773d9b6f278"
```

Comment:
As mentioned, this shouldn't have changed - I imagine we need to revert `royalties.wasm` etc